nuget/lib/dependabot/nuget/native_helpers.rb

```diff
@@ -103,14 +97,30 @@ def self.run_nuget_updater_tool(repo_root:, proj_path:, dependency:, is_transiti
           "--verbose"
         ].compact.join(" ")
 
+        [command, fingerprint]
+      end
+
+      sig do
+        params(
+          repo_root: String,
+          proj_path: String,
+          dependency: Dependency,
+          is_transitive: T::Boolean,
+          credentials: T::Array[T.untyped]
```

Comment:
Can you double check if this typing works now?

```suggestion
          credentials: T::Array[Dependabot::Credential]
```

I think whenever I added this initially, types were still a little up in the air.